public/app/features/alerting/unified/components/rule-editor/QueryWrapper.tsx

```diff
@@ -58,18 +58,34 @@ export const QueryWrapper: FC<Props> = ({
   const isExpression = isExpressionQuery(query.model);
   const [pluginId, changePluginId] = useState<SupportedPanelPlugins>(isExpression ? TABLE : TIMESERIES);
 
-  const renderTimePicker = (query: AlertQuery, index: number): ReactNode => {
-    if (isExpressionQuery(query.model) || !onChangeTimeRange) {
-      return null;
-    }
-
+  function SelectingDataSourceTooltip() {
+    const styles = useStyles2(getStyles);
     return (
-      <RelativeTimeRangePicker
-        timeRange={query.relativeTimeRange ?? getDefaultRelativeTimeRange()}
-        onChange={(range) => onChangeTimeRange(range, index)}
-      />
+      <div className={styles.dsTooltip}>
+        <Tooltip content={<>Not finding the data source you want? Some data sources are not supported for alerting.</>}>
```

Comment:
@brendamuir Maybe it's worth revising the user-facing messages to provide as much information as available